src/ml/neural_net/compute_context.hpp

```diff
@@ -16,6 +16,19 @@
 namespace turi {
 namespace neural_net {
 
+/** A struct to define all the parameters used to create the activity classifier
+ * model backend
+ */
+struct ac_parameters {
+  int batch_size;
+  int num_features;
+  int prediction_window;
+  int num_classes;
+  int num_predictions_per_chunk;
+  int random_seed;
```

Comment:
For these let's specify reasonable default values, as in `int batch_size = 32;` or whatever the actual default is.

(And add documentation)

Reply:
The default values come from `init_options`. For example, `batch_size` is already 32 in it.